cypress/integration/consign.spec.ts

```diff
@@ -14,4 +14,24 @@ describe("Consign", () => {
       "Sell Art from Your Collection | Consignments | Artsy"
     )
   })
+  it("CTA buttons appear after the hero section scrolled", () => {
```

Comment:
@iskounen Added integration test to check sticky behavior

Reply:
This test looks great 🙌